theme/mixins/backdrop.less

```diff
@@ -1,9 +1,13 @@
 .transparentBackdrop() {
   position: fixed;
-  top: -999px;
-  left: -999px;
-  right: -999px;
-  bottom: -9999px;
+  top: 0;
+  left: 0;
+  // calc(100vw -1) is an adjustment for FF because it adheres to w3c spec
+  // in that it includes the scrollbars. This however is problematic
+  // for us as it means there is still a small space that overflows:
+  // https://bugzilla.mozilla.org/show_bug.cgi?id=811403
+  width: calc(100vw - 1);
```

Comment:
Why a unit-less one? Does the adjustment need to be unit-less or could we specify this as a pixel?

Reply:
We may actually take a slightly different approach, but my brain is fried today (I need to verify it integration side e.g. chalice). So will try it out tomorrow morning and see if the approach Mike mentioned above works. It'd be simpler and generally better IMHO.